src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -29,6 +29,7 @@
     <ProjectCapability Include="ManagedLang"/> <!-- Temporary: See https://github.com/dotnet/roslyn-project-system/issues/47 -->
     <ProjectCapability Include="HandlesOwnReload"/>
     <ProjectCapability Include="UseFileGlobs"/>
+    <ProjectCapability Include="ManagedEditAndContinue"/>
```

Comment:
There has been a mistake. I did not notice the constituent projects in the sample solution I was working with. I mistook the EnC component load of a .Net Core project for the shared project.

This EnC component is loaded with the capability to load only for the Managed projects so it won't load for shared projects.